packages/components/src/local/atoms/heartbeat-checker/index.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+
+// Import component files
+import HeartbeatChecker from './index'
+import PropTypes from './types/props'
+import docs from './README.md'
+import { Story } from '@storybook/react/types-6-0'
+
+export default {
+  title: 'local/atoms/HeartbeatChecker',
+  component: HeartbeatChecker,
+  decorators: [],
+  parameters: {
+    readme: {
+      // Show readme before story
+      content: docs
+    }
+  },
+  argTypes: {
+    enableHeartbeat: {
+      control: {
+        type: 'boolean'
+      }
+    }
+  }
+}
+
```

Comment:
@sanjayradadiya - do we have a background server running for StoryBook tests?  So, does the heartbeat work for this?

IMHO, we should mock the backend call, and instead let a toggle control if  (or what) is returned from the `healthcheck` URL.

Reply:
this is plain component it will only animate when flag enabled